backend/src/test/java/com/github/lenaruppert/backend/controller/ClientControllerTest.java

```diff
@@ -34,17 +36,19 @@ void setUp() {
 
     @Test
     @DirtiesContext
+    @WithMockUser(username = "user", password = "password")
```

Comment:
```suggestion
    @WithMockUser
```
If you mock user is called `user`, you can just use the annotation without anything in the brackets (applies to the other tests too).